public/js/new-user.js

```diff
@@ -0,0 +1,18 @@
+let users = [];
+
+class NewUser {
```

Comment:
Class could just be called 'Users', and addUser could be a method inside this users class